src/main/resources/zenodo-1.0.0-swagger-2.0.yaml

```diff
@@ -524,13 +524,73 @@ paths:
         - preview
       produces:
         - application/json
+      parameters:
+        - in: query
+          name: q
+          type: string
+          description: 'Search query (using Elasticsearch query string syntax)'
+        - in: query
+          name: sort
+          type: string
+          enum:
+            - "bestmatch"
+            - "newest"
+            - "oldest"
+            - "updated-desc"
+            - "updated-asc"
+            - "version"
+            - "mostviewed"
+            - "mostdownloaded"
+        - in: query
+          name: page
+          type: integer
+        - in: query
+          name: size
+          type: integer
       responses:
         '200':
           description: OK
           schema:
-            type: object
+            $ref: '#/definitions/SearchResult'
         '405':
           description: Method Not Allowed
+  /records/{recordId}:
+    get:
+      summary: Get a single record
+      parameters:
+        - in: path
+          name: recordId
+          type: integer
```

Comment:
The [docs](https://inveniordm.docs.cern.ch/reference/rest_api_drafts_records/#get-a-record) define it as a string although I think we've been using integers for IDs elsewhere too